source/core/Native.cpp

```diff
@@ -282,5 +282,104 @@ namespace GTA
 
 			return static_cast<T>(ObjectFromNative(T::typeid, task->_result));
 		}
+
+		int GetUtf8CodePointSize(System::String ^string, int index)
+		{
+			unsigned int chr = static_cast<unsigned int>(string[index]);
+
+			if (chr < 0)
```

Comment:
An 'unsigned int'  cannot be less than zero, so this branch is never executed =)